ibrahim/codepen.html

```diff
@@ -0,0 +1,170 @@
+<!-- Copied from https://codepen.io/ibnoon/pen/MWwqYJQ?editors=1000  -->
+
+<!-- MAX.G Logo -->
+<img class="MaxG" src="http://angeliqueweger.com/temp/gd2-images/ibrahim/inoon_small_icon.svg" alt="Max Gallery">
+
+<img class="MaxG_l" src="http://angeliqueweger.com/temp/gd2-images/ibrahim/inoon_banner.svg" alt="Max Gallery">
+
+<!-- Nav -->
+<nav>
+  <ul class="nav">
+    <li><a href="#">About</a></li>
+    <li><a href="#">Event</a></li>
+    <li><a href="#">Contact</a></li>
+  </ul>
+</nav>
+
+<nav>
+  <ul class="nav_l">
+    <li><a href="#">About</a></li>
+    <li><a href="#">News</a></l>
+    <li><a href="#">Event</a></li>
+    <li><a href="#">Exhibition</a></li>
+    <li><a href="#">Publications</a></li>
+    <li><a href="#">Contact</a></li>
+  </ul>
+</nav>
+
+<!-- Header Image -->
+<img src="http://angeliqueweger.com/temp/gd2-images/gallery3.jpg" alt="">
+
+<div class="body">
```

Comment:
Maybe instead of a classname `body`, you can replace this `div` with a `main` element? This would eliminate a confusing classname and incorporate a more meaningful HTML element.